docs/virtual_machines/guest_agent_information.md

```diff
@@ -40,16 +40,21 @@ GA info with merged into status
             version: "29"
             versionId: "29"
         interfaces:
-            - interfaceName: eth0
-            ipAddress: 10.244.0.23/24
-            ipAddresses:
-                - 10.244.0.23/24
-                - fe80::858:aff:fef4:17/64
-            mac: 0a:58:0a:f4:00:17
-            name: default
-
-When the Guest Agent is not present in the Virtual Machine, the information is not shown. No error is reported because the Guest Agent is an optional component.
-
+            - infoSource: domain, guest-agent
+              interfaceName: eth0
+              ipAddress: 10.244.0.23/24
+              ipAddresses:
+              - 10.244.0.23/24
+              - fe80::858:aff:fef4:17/64
+              mac: 0a:58:0a:f4:00:17
+              name: default
+
+When the Guest Agent is not present in the Virtual Machine, the Guest Agent information is not shown. No error is reported because the Guest Agent is an optional component.
+
+The infoSource field indicates where the info is gather from. Valid values:
+1. domain: the info is based on the domain spec
+2. guest-agent: the info is based on Guest Agent report
+3. domain, guest-agent: the info is based on both the domain spec and the Guest Agent report
```

Comment:
For some reason, this is not ending up well formatted on the UI. Check the preview.